src/coreclr/vm/arm/asmhelpers.S

```diff
@@ -127,7 +127,8 @@ LOCAL_LABEL(LReturnDone):
         EPILOG_STACK_RESTORE_OFFSET   r7, #8
         EPILOG_POP              "{r4,r5,r7,pc}"
 
-PATCH_LABEL CallDescrWorkerInternalReturnAddressOffset
+CallDescrWorkerInternalReturnAddressOffset:
+    .global CallDescrWorkerInternalReturnAddressOffset
```

Comment:
I assume that this is to get rid of the Thumb bit, right? Are there any more places with `PATCH_LABEL` where this could be an issue?

Reply:
Looks like the only remaining usage is `ExternalMethodFixupPatchLabel` and presumably that one is fine. It goes through the `GetEEFuncEntryPoint` macro and that macro enforces the Thumb bit anyway.

Reply:
There is no other label to data that would suffer from this problem. All the other using PATCH_LABEL are pointing to code, so they are correct with the thumb bit.